### PR TITLE
add packages needed for curb and rmagick

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: appengine/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 
 # Install dependencies for Ruby
 # Also installs dependencies for the following common gems:
-# nokogiri, sqlite3, mysql2, pg
+# nokogiri, sqlite3, mysql2
+# 
+# gems      dependencies
+# ------------------------------------------------------------------
+# curb      libcurl3, libcurl3-gnutls, libcurl4-openssl-dev
+# pg        libpq
```
Comment (dazuma) [inline appengine/Dockerfile]: libpq-dev

File: appengine/Dockerfile [hunk 2]
```diff
@@ -1,11 +1,17 @@
 # Base Dockerfile for an App Engine Ruby runtime.
 # Dockerfiles generated by the gcloud tool inherit FROM this image.
-
 FROM gcr.io/google_appengine/base
 
 # Install dependencies for Ruby
 # Also installs dependencies for the following common gems:
-# nokogiri, sqlite3, mysql2, pg
+# nokogiri, sqlite3, mysql2
```
Comment (dazuma) [inline appengine/Dockerfile]: Why not list these gems in your table as well:
nokogiri: libxml2-dev
sqlite3: libsqlite3-dev
mysql2: libmysqlclient-dev

**Comment by JustinBeckwith** [comment]
Good call - done

File: appengine/Dockerfile (hunk 2 above)
Comment (dazuma) [inline appengine/Dockerfile]: And we can get rid of this line. :)

**Comment by JustinBeckwith** [comment]
done

**Comment by dazuma** [comment]
LGTM

**Pull request merged**